src/isIsogram.test.js

```diff
@@ -6,4 +6,16 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should return false if entered string is lowercase', () => {
```

Comment:
But it will return true for the lowercase isogram. It will return false only with a repetition - l**oo**k

Reply:
but then more than 80 symbols and error, I can't think today anymore(((( Then this wirk I will skip for now

Reply:
We discussed it during the Q&A, I thought you would recall it.

A solution, called concatenation:
```
it('should return false if entered string'
    + ' is lowercase and has repeatings', () => {
```